src/main/java/org/folio/des/service/FolioTenantService.java

```diff
@@ -55,6 +76,9 @@ protected void afterTenantUpdate(TenantAttributes tenantAttributes) {
       oldJobDeleteScheduler.scheduleOldJobDeletion(context.getTenantId());
       kafka.createKafkaTopics();
       kafka.restartEventListeners();
+
+      bursarMigrationService.updateLegacyBursarJobs(bursarExportLegacyJobService, jobService);
```

Comment:
this will be executed in all further releases even when already migrated? do you plan to remove this code or how will you manage migration process?

Reply:
IMO this should probably stay in place, at least for the foreseeable future.  All other types of migration actions (liquibase updates, RMB scripts, etc) are always checked on upgrade — there's nothing that prevents someone from updating this module from v1.0.0 to v999 or any other combination, and we should be able to handle any version range without breaking, meaning these migration scripts would need to stick around for some time.